Sim_Harness_GUI/obj/Debug/Sim_Harness_GUI.gtk-gui.gui.stetic

```diff
@@ -7,22 +7,22 @@
     <widget-library name="glade-sharp, Version=2.12.0.0, Culture=neutral, PublicKeyToken=35e10195dab3c99f" />
```

Comment:
Hold on don't touch this branch I am making some changes to it. I might be making a new and improved brach to fix some of this code.